source/index.rst

```diff
@@ -28,6 +28,8 @@ Special Thanks
 
 These are institutions who were early adopters or provided HPC resources for development and testing of the resource manager adapters currently available. The success of this project would not be possible without their generous support (in alphabetical order):
 
+- Bowdoin College. For early adoption and feedback with the SGE adapter.
+- Oregon State University. For early adoption and feedback with the SGE adapter.
```

Comment:
I was considering "early adoption" to be "early adoption of Open OnDemand" as a platform i.e. the time before we had an rpm and had zero installs. I wanted to call out the first few institutions who dived in when they knew they were the first ones to use it.

Reply:
ah...up to you. I'm not too worried about the wording.